cmake/FindOpenSSL.cmake

```diff
@@ -1,136 +0,0 @@
-include( FindPackageHandleStandardArgs )
-
-set( OPENSSL_GOOD_VERSION TRUE )
-if( OPENSSL_INCLUDE_DIR AND OPENSSL_LIBRARIES )
-  set( OPENSSL_FOUND TRUE )
-else()
-  find_path(
-    OPENSSL_INCLUDE_DIR
-    NAMES openssl/ssl.h
-    HINTS
-    ${OPENSSL_ROOT_DIR}
-    PATH_SUFFIXES
-    include )
-
-  file(STRINGS ${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h opensslvers REGEX "^# define OPENSSL_VERSION_NUMBER")
-  string(REGEX REPLACE "# define[ ]+OPENSSL_VERSION_NUMBER[ ]+" "" opensslvers ${opensslvers})
-
-  set( LIBSSLNAME ssl )
-  set( LIBCRYPTONAME crypto )
-
-  if ( OPENSSL_GOOD_VERSION )
-
-    find_library(
-      OPENSSL_SSL_LIBRARY
-      NAMES ${LIBSSLNAME}
-      HINTS
-      ${OPENSSL_ROOT_DIR}
-      PATH_SUFFIXES
-      ${LIBRARY_PATH_PREFIX}
-      ${LIB_SEARCH_OPTIONS})
-
-    string(FIND ${OPENSSL_SSL_LIBRARY} ".a" hasstaticext)
-    if (NOT ${hasstaticext} EQUAL -1)
-      if ( ${opensslvers} LESS "0x1000201fL" )
-        set( OPENSSL_GOOD_VERSION FALSE )
-        message(WARNING " >>> Cannot build XRootD crypto modules: static openssl build version is < 1.0.2")
```

Comment:
This check was the reason for me to add the minimum version to `find_package(OpenSSL)`.